### ISSUE TITLE
Body Representation breaks if climbing through geometry

### ISSUE THREAD

**Issue by bddckr** [ExtendRealityLtd/VRTK.Unity.Core#180]
#157 brought Body Representation and Climbing.

If the user climbs down and therefore moves the play area through a collider the body won't follow as the virtual physics prevent it from doing so. While this is the expected and desired outcome the Body Representation logic actually breaks when doing this - the play area offset from the body staying out of the geometry keeps being applied to the body.

This problem ultimately boils down to a regular event of body ↔️ headset divergence which should probably be handled by default for this particular issue. A solution currently is to disable and enable the 
`BodyRepresentationInternalSetup` as its `OnEnable` ensures the necessary movement of the play area to converge the headset back into the virtual body.

**Comment by thestonefox** [comment]
Is this still an issue?

**Comment by bddckr** [comment]
Yeah it's part of my 5 bullet point list to enhance climbing further. The idea is to not allow diverging, even while climbing, by utilizing a smaller collider (that resembles the headset rather than the whole body).

**Comment by pushmatrix** [comment]
Would this cause the character to release the grab when the collision happens?

I'm encountering a scenario where if my body touches a collider while climbing, the grab lets go. Wondering if it's related.
